projects/jsf/src/lib/validator.service.ts

```diff
@@ -57,6 +58,45 @@ export class ValidatorService {
       validators.push(Validators.minLength(options.length.minLength));
     }
 
+    if (options.pattern) {
+      validators.push(Validators.pattern(options.pattern));
+    }
+
+    return validators;
+  }
+
+  private getIntegerValidators(item: IntegerDataItem): ValidatorFn[] {
+    const validators = [];
+    const options = item.validationSettings;
+
+    validators.push(this.getIntValidator());
+    validators.push(Validators.min(ValidatorService.MIN_NUMBER)); // may be overwritten below
+    validators.push(Validators.max(ValidatorService.MAX_NUMBER)); // may be overwritten below
+
+    if (options.length?.maxLength) {
+      validators.push(Validators.maxLength(options.length.maxLength));
+    }
+
+    if (options.length?.minLength) {
+      validators.push(Validators.minLength(options.length.minLength));
+    }
```

Comment:
Is this a copy/paste error? I couldn't find this in the docs.

Reply:
When I broke integer out of the string class, these keywords were not excluded for integer fields. I didn't want to break anything, but you're right, they are not part of the JSON schema definition for integers